WooCommerce/Classes/ViewRelated/Orders/Order Creation/OrderForm.swift

```diff
@@ -339,6 +371,11 @@ private extension OrderForm {
                                                                    comment: "Title text of the button to add a single product via SKU scanning")
         static let productRowAccessibilityHint = NSLocalizedString("Opens product detail.",
                                                                    comment: "Accessibility hint for selecting a product in an order form")
+        static let permissionsTitle =
+        NSLocalizedString("Camera permissions", comment: "Title of the action sheet button that links to settings for camera access")
+        static let permissionsMessage = NSLocalizedString("Please change your camera permissions in device settings",
```

Comment:
This string could use some work, it's not very descriptive of why they need to give permission, or what we'll use it for. Perhaps it would be best to use the same string as we define in the Plist which shows when the permissions request (from .notDetermined) is invoked? Or something specific to SKU scanning.

Reply:
Definitely, changed the wording here https://github.com/woocommerce/woocommerce-ios/pull/9687/commits/18863ad72794dd37cee36d31c855972103fd20e9 to be more specific about barcode scanning